### PR TITLE
Remove unused glyphs and Macintosh table

### PR DESCRIPTION
**Pull request by ylemkimon**
Glyphs U+EFFD through U+EFFF (included in the base font `Space.ttx`) are not used.

Also, removed Macintosh table:
> Note that the use of the Macintosh platformID is currently discouraged. Subtables with a Macintosh platformID are only required for backwards compatibility with QuickDraw and will be synthesized from Unicode-based subtables if ever needed.

https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6cmap.html

### CODE REVIEW

File: src/fonts/lib/Space.ttx [hunk 1]
```diff
@@ -93,8 +91,8 @@
     <ulUnicodeRange4 value="00000000 00000000 00000000 00000000"/>
     <achVendID value="PfEd"/>
     <fsSelection value="00000000 01000000"/>
-    <usFirstCharIndex value="61438"/>
```
Comment (kevinbarabash) [inline src/fonts/lib/Space.ttx]: Was the `usFirstCharIndex` wrong?

File: src/fonts/lib/Space.ttx [hunk 2]
```diff
@@ -110,46 +108,6 @@
   </OS_2>
 
   <name>
-    <namerecord nameID="0" platformID="1" platEncID="0" langID="0x0" unicode="True">
-      Copyright (c) 2009-2010 Design Science, Inc.
-Copyright (c) 2014-2018 Khan Academy
-    </namerecord>
```
Comment (kevinbarabash) [inline src/fonts/lib/Space.ttx]: `platformID="1"` is the Macintosh platform which is why we're getting rid of all of these.

File: src/fonts/lib/Space.ttx [hunk 3]
```diff
@@ -191,20 +149,10 @@ http://scripts.sil.org/OFL
...
-      <map code="0xeffe" name="uniEFFE"/><!-- ???? -->
-      <map code="0xefff" name="uniEFFF"/><!-- ???? -->
     </cmap_format_4>
-    <cmap_format_0 platformID="1" platEncID="0" language="0">
-      <map code="0x9" name="space"/>
-      <map code="0xd" name="space"/>
-      <map code="0x20" name="space"/>
-      <map code="0xca" name="uni00A0"/>
-    </cmap_format_0>
     <cmap_format_4 platformID="3" platEncID="1" language="0">
```
Comment (kevinbarabash) [inline src/fonts/lib/Space.ttx]: Would it make sense to get rid of Windows specific stuff in a separate PR or do we need this?

File: src/fonts/lib/Space.ttx [hunk 4]
```diff
@@ -48,12 +46,12 @@
     <reserved2 value="0"/>
     <reserved3 value="0"/>
     <metricDataFormat value="0"/>
-    <numberOfHMetrics value="5"/>
+    <numberOfHMetrics value="1"/>
   </hhea>
```
Comment (kevinbarabash) [inline src/fonts/lib/Space.ttx]: Why did this change from `5` to `1`?  Was the old value wrong?

File: src/fonts/lib/Space.ttx [hunk 5]
```diff
@@ -236,7 +184,7 @@ http://scripts.sil.org/OFL
       <PaintType value="0"/>
       <CharstringType value="2"/>
       <FontMatrix value="0.001 0 0 0.001 0 0"/>
-      <FontBBox value="0 0 300 533"/>
+      <FontBBox value="50 0 200 533"/>
       <StrokeWidth value="0"/>
```
Comment (kevinbarabash) [inline src/fonts/lib/Space.ttx]: This changed because of the removal of `0xeffe` and `0xefff`.  How did you determine the new values?

**Comment by ylemkimon** [comment]
@kevinbarabash These values are automatically generated by compiling and decompiling `Space.ttx`:
```bash
ttx Space.ttx
ttx Space.otf
```

File: src/fonts/lib/Space.ttx (hunk 3 above)
Comment (ylemkimon) [reply src/fonts/lib/Space.ttx]: @kevinbarabash I haven't checked but I guess they are required in the Internet Explorer.

Comment (ylemkimon) [reply src/fonts/lib/Space.ttx]: [Microsoft docs](https://docs.microsoft.com/ko-kr/typography/opentype/spec/cmap) state:
> Microsoft **strongly** recommends using Unicode 'cmap' subtables for all fonts.

I'm not sure whether it means we should use `platformID="3" platEncID="1"` or not use `platformID="3"` at all.

Comment (kevinbarabash) [reply src/fonts/lib/Space.ttx]: I guess leave it for now until we find something more conclusive.

**Pull request merged**